back-end/__tests__/endpoints.test.js

```diff
@@ -0,0 +1,5 @@
+describe('setup', () => {
+  it('Testing that endpoints are posting data', () => {
+    expect(1).toBe(1);
+  });
+});
```

Comment:
Usually mocks are in such `__tests___` folder. A regular application test should probably be located in `back-end/tests`

Reply:
noted. I'll adjust at next PR

Reply:
Actually, it seems like some projects out there are using that `__tests__` folder. It's just that I haven't seen that often at all :/